content/en/docs/spinnaker-user-guides/pacrd.md

```diff
@@ -458,6 +458,134 @@ Events:
   Warning  PipelineValidationFailed  0s (x4 over 3s)        pipelines  artifact with id "a-nonsense-value" and name "" could not be found for this pipeline
 ```
 
+
+## Setup mTLS
+
+### Pre-requisites
+For setting up mTLS with PaCRD you need the following:
+- ca.pem file
+- ca.key file
+- ca password
+
+Once you have that information you can continue.
```

Comment:
Maybe modify this to say that if you don't have them follow the full setup instructions, otherwise skip to step 2?

Reply:
I'm confused. So it looks like I need `ca.pem`, `ca.key`, and the ca password. In addition, I have to generate specific certificates for pacrd using the included bash script.

So, if I have a brand new installation, I should follow the instructions in "Generating Certificates" to generate all the certs I need, including the pacrd ones.  If I'm using an existing installation, I should use the script below to generate only the pacrd certs.  ???

Reply:
I just changed the instructions to reflect a little more the details regarding the process. If you don't have those files is because you haven't setup mTLS and you need to follow the standard process to setup mTLS. If you have them and you don't have the pacrd you can use the script.